index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page Built by Sheldon and Muda</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Nelson Mandela</h1>
+        <div id="img-div">
+            <img class="image" src="image/pexels-minenhle-shelembe-5985865.jpg" alt="Nelson Mandela">
+            <figcaption id="img-caption">Picture of Nelson Mandela</figcaption>
+        </div>
+        <div class="tribute-info" >
+            <p>About Nelson madela</p>
+            <a class="info" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target="_blank">Read More about our hero Nelson Mandela</a>
+        </div>
+
+
+    </main>
```

Comment:
I would suggest you align your code properly for easier reading, maintaining, and debugging.